instance/ssh.py

```diff
@@ -0,0 +1,23 @@
+"""
+Utilities for dealing with SSH.
+"""
+import logging
+import subprocess
+
+LOGGER = logging.getLogger(__name__)
+
+
+def remove_known_host_key(ip: str):
+    """
+    Removes the known_hosts key entry for the given IP.
+
+    :param ip: the IP of the entry to delete
+
+    This doesn't check whether the entry existed.
+    """
+    LOGGER.info('Deleting the SSH host key of the server with the IP address "%s" from the SSH known hosts file.', ip)
```

Comment:
@kewne, when testing this, I see a lot of log lines that say

```
Deleting the SSH host key of the server with the IP address "None" from the SSH known hosts file.
```

Can't we just not run code in this function when the IP is `None`?

Reply:
I'm resolving this because the issue already existed. I'll create a follow-up story, it seems like a particularly newcomer friendly issue.